Armadillo/src/main/java/com/scribd/armadillo/playback/ExoPlaybackExceptionExt.kt

```diff
@@ -30,9 +30,8 @@ internal fun ExoPlaybackException.toArmadilloException(): ArmadilloException {
                     val httpCause = source.cause as? HttpDataSource.InvalidResponseCodeException
                     HttpResponseCodeException(httpCause?.responseCode ?: 0, httpCause?.dataSpec?.uri.toString(), source)
                 }
+                is UnknownHostException,
```

Comment:
The reason this was mapped to an http exception was so that we could have the host in the error for tracking. Any way to add that here?

Reply:
This exception appears in airplane mode, so it might not accurately reflect whether the host exists in DNS. Is it still valuable in that case?

Reply:
Yeah, that's a good point. It's probably fine.